src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -117,132 +117,30 @@
 # import boto sdb connection for autoscaling
 import boto.sdb
 
+import textwrap
```

Comment:
Will do. Seem preferable to my method, which is usually "the order in which I realized I needed that module"